doc/classes/SurfaceTool.xml

```diff
@@ -66,6 +69,7 @@
 			</argument>
 			<description>
 				Specifies whether the current vertex (if using only vertex arrays) or current index (if also using index arrays) should use smooth normals for normal calculation.
+				[b]Note:[/b] This must be called [i]before[/i] calling [method add_vertex]. Otherwise, it won't behave as desired.
```

Comment:
I think this can be called when creating the index array. Which means it can be called after all calls to add_vertex